src/get-started/test-service.md

```diff
@@ -100,13 +99,13 @@ To perform testing, we first need to create a network emulation – the eponymou
 Since `TestKit` will be used by all tests, it is natural to move its
 constructor to a separate function:
 
+<!-- markdownlint-disable MD013 -->
 ```rust
 fn init_testkit() -> TestKit {
-    TestKitBuilder::validator()
-        .with_service(CurrencyService)
-        .create()
+    TestKit::for_rust_service(CryptocurrencyService, INSTANCE_NAME, INSTANCE_ID, ())
```

Comment:
Could make sense to define constants beforehand and explain what they mean; currently, they appear out of nowhere here.